Src/Base/AMReX_BLBackTrace.cpp

```diff
@@ -188,9 +189,29 @@ BLBackTrace::print_backtrace_info (FILE* f)
                 // cmd = "/usr/bin/eu-addr2line -C -f -i --pretty-print -p "
                 cmd = "/usr/bin/eu-addr2line -C -f -i -p "
                     + std::to_string(pid);
+
+                for (int i = 0; i < nentries; ++i)
+                {
+
+#if !defined(AMREX_USE_OMP) || !defined(__INTEL_COMPILER)
+                  std::string addr2line_result;
+                  if (amrex::system::call_addr2line && have_eu_addr2line) {
+                    if (bt_buffer[i] != nullptr) {
+                      char print_buff[32];
+                      std::snprintf(print_buff,sizeof(print_buff),"%p",bt_buffer[i]);
+                      const std::string full_cmd = cmd + " " + print_buff;
+                      addr2line_result = run_command(full_cmd);
+                    }
+                  }
+                  if (addr2line_result.find("AMReX_BLBackTrace.cpp:") != std::string::npos) {
+                    use_eu_addr2line = 1;
+                    break;
```

Comment:
We should not break.

Reply:
If the test is passed, the entry is good.  So we want to keep printing more, not break.

Reply:
Let me checkout the code.  Reading diff is hard.

Reply:
It sounds like it would be better to store both the command strings and make the test for BLBacktrace for eu-addr2line be inside the for loop which is printing, rather than trying to check it ahead of printing in another for loop. Copying the code like I did here might be less readable/maintainable